README.md

```diff
@@ -83,6 +83,10 @@ AIRTABLE_IMPORT_SETTINGS = {
         # If you choose option #1 (callable) or option #2 (path to a function)
         # Your function needs to return an integer which will represent the Parent
         # Page ID where all imported pages will be created as child pages.
+        # The path to a function takes an `instance` parameter such as
```

Comment:
This is a bit confusingly worded - presumably the path to a function is just a string, and it's instead the function itself (or the callable? Shouldn't this work for option 1 as well?) that takes the `instance` kwarg?

Reply:
You are super correct, that is confusing. I'll re-write this. Thanks for the review Jacob!